frontend/src/tests/sightings/SightingPage.test.tsx

```diff
@@ -0,0 +1,79 @@
+import { render, screen } from "@testing-library/react";
+import { createMemoryHistory } from "history";
+import { Router } from "react-router-dom";
+import { SightingPage } from "../../components/SightingPage/SightingPage";
+import * as apiClient from "../../clients/internalApiClient";
+import { act } from "react-dom/test-utils";
```

Comment:
Please instead import `act` from the import on line 1, `@testing-library/react`

Reply:
Oh sorry I wasn't clear, I wanted you to move this import to be inside the import on line 1, so your line 1 should be
```
import { render, screen, act } ...
```